cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -1,24 +1,123 @@
-    <entry issue="ESMCI/cime#2917" >FAIL PFS_Ld20.f09_g17.I2000Clm50BgcCrop.cheyenne_intel BASELINE</entry>
-  </category>
-  <category name="fates">
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.hobart_nag.clm-Fates COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERP_Ld9.f45_f45_mg37.I2000Clm45Fates.hobart_nag.clm-FatesAllVars COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.cheyenne_intel.clm-FatesLogging COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.cheyenne_intel.clm-Fates COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.cheyenne_intel.clm-FatesNoFire COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.cheyenne_intel.clm-FatesPPhys COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.cheyenne_intel.clm-FatesST3 COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERP_Ld9.f45_f45_mg37.I2000Clm45Fates.cheyenne_intel.clm-FatesAllVars COMPARE_base_rest</entry>
-    <entry issue="NGEET/fates#315">FAIL ERS_Ld60.f45_f45_mg37.I2000Clm45Fates.cheyenne_gnu.clm-Fates COMPARE_base_rest</entry>
-  </category>
+<expectedFails version="1.0">
+
+  <!-- aux_clm test suite failures -->
+
+  <test name="ERP_D_Ld10_P36x2.f10_f10_musgs.IHistClm50BgcCrop.cheyenne_intel.clm-ciso_decStart">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>#404</issue>
+    </phase>
+    <phase name="COMPARE_base_rest">
+      <status>PEND</status>
+      <issue>#404</issue>
+      <comment>Because of the RUN failure, this phase is listed as PEND</comment>
+    </phase>
+  </test>
+
+  <test name="ERS_Lm20_Mmpi-serial.1x1_smallvilleIA.I2000Clm50BgcCropGs.cheyenne_gnu.clm-monthly">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>#158</issue>
+    </phase>
+  </test>
+
+  <test name="SMS.f10_f10_musgs.I2000Clm50BgcCrop.hobart_pgi.clm-crop">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>#442</issue>
+    </phase>
+  </test>
+
+  <test name="SMS_D.f10_f10_musgs.I2000Clm50BgcCrop.hobart_pgi.clm-crop">
+    <phase name="RUN">
+      <status>FAIL</status>
+      <issue>#442</issue>
+    </phase>
+  </test>
+
+  <test name="ERS_D_Ln9_P480x3.f19_g16.I2000Clm50SpGs.cheyenne_intel.clm-waccmx_offline">
+    <phase name="COMPARE_base_rest">
+      <status>FAIL</status>
+      <issue>#550</issue>
+    </phase>
+  </test>
+
+  <test name="PFS_Ld20.f09_g17.I2000Clm50BgcCrop.cheyenne_intel">
+    <phase name="GENERATE">
+      <status>FAIL</status>
+      <issue>ESMCI/cime#2917</issue>
+    </phase>
+    <phase name="BASELINE">
+      <status>FAIL</status>
+      <issue>ESMCI/cime#2917</issue>
+    </phase>
+  </test>
+
+  <!-- fates test suite failures -->
```

Comment:
Note that different test suites are now delineated by comments, not by xml sections. This is because, in the machine parsing of this file, we don't have an easy way to determine which category is appropriate, and in any case, it's not really important: if a test from one category is supposed to fail, then it will fail regardless of how we're running the test – including if we're running individual tests, not part of any test suite at all.

Reply:
I'd kind of like to add an optional field that gives the test name. I don't think I even care if it isn't read in now, but I think it helps to organize this for humans.

`<category>fates</category>`

I propose you just enter it for the non aux_clm tests. But, it wouldn't be bad to put for all of them. The comments technically also take care of this, but I like making it explicit in the XML.

So I'd say this is a preference. I think it will help in the future. But, it's not a must have.

Reply:
Okay, take a look at the commit I just pushed and let me know if that's what you were thinking. Are you happy with this other than that, or do you still want to look some more?

Reply:
This is great. Thanks for doing that, as well as adding the header at the top.